packages/cli/src/validators/getUnappliedSchemaOverridesTableNames.ts

```diff
@@ -0,0 +1,33 @@
+import { SchemaDefOverrides } from '../types/CliConfig';
+
+export function getUnappliedSchemaOverridesTableNames(
+    tables: string[],
+    defaultSchema: string,
+    schemaDefOverrides?: SchemaDefOverrides,
+) {
+    const unappliedSchemaOverrideTableKeys: string[] = [];
+
+    if (isObject(schemaDefOverrides)) {
+        const tablesToOverride = Object.keys(schemaDefOverrides);
+
+        for (const tableKey of tablesToOverride) {
+            const [schemaName, tableName] = tableKey.split('.');
+
+            if (schemaName === defaultSchema) {
+                if (!tables.includes(tableName)) {
+                    unappliedSchemaOverrideTableKeys.push(`- ${tableKey}`);
```

Comment:
It's really weird that this function returns an array of `- table`. It completely breaks the potential composability of this function.

I would suggest you remove the `-` part.

It's like mixing two concerns into one: the rendering (`- `) and the logic (finding unused overrides).

Reply:
Yeah, for sure. there is essentially _zero_ cost to iterating what will in the worst case be a few 1000 items.